nirps/manual_trigger/manual_trigger.py

```diff
@@ -751,7 +751,7 @@ def run_lbl_processing(settings: Dict[str, Any]):
         run_apero_reduction_interface(trigger_settings)
         # deal with only running processing
         if trigger_settings['ONLY_REDUCTIONINTERFACE']:
-            print_process('Only running apero get')
+            print_process('Only running ARI')
```

Comment:
This fixes a typo. Not directly related to everything else but should be merged either here or separately.